--- old/dosnet.inf	2007-02-05 14:23:58.000000000 +0000
+++ new/dosnet.inf	2007-02-05 14:33:21.000000000 +0000
@@ -545,6 +545,27 @@
 
 
 [FloppyFiles.2]
+d1,FASTTX2K.sys
+d1,INIC162X.sys
+d1,AARICH.sys
+d1,NVATA.sys
+d1,NVATABUS.sys
+d1,NVRAID.sys
+d1,Si3112r.sys
+d1,vmscsi.sys
+d1,adp94xx.sys
+d1,ultra.sys
+d1,megasas.sys
+d1,symmpi.sys
+d1,ql2300.sys
+d1,ql40xx.sys
+d1,perc2.sys
+d1,adpu160m.sys
+d1,mraid2k.sys
+d1,nfrd960.sys
+d1,a320raid.sys
+d1,iastor.sys
+d1,aacsas.sys
 d1,disk1,disk103
 d1,cpqarray.sys
 d1,ncrc710.sys
@@ -612,6 +633,27 @@
 ntldr
 
 [Files]
+d1,FASTTX2K.sys
+d1,INIC162X.sys
+d1,AARICH.sys
+d1,NVATA.sys
+d1,NVATABUS.sys
+d1,NVRAID.sys
+d1,Si3112r.sys
+d1,vmscsi.sys
+d1,adp94xx.sys
+d1,ultra.sys
+d1,megasas.sys
+d1,symmpi.sys
+d1,ql2300.sys
+d1,ql40xx.sys
+d1,perc2.sys
+d1,adpu160m.sys
+d1,mraid2k.sys
+d1,nfrd960.sys
+d1,a320raid.sys
+d1,iastor.sys
+d1,aacsas.sys
 d1,usetup.exe,system32\smss.exe
 d1,ntdll.dll,system32\ntdll.dll
 
